addons/point_of_sale/static/src/app/screens/product_screen/product_list/product_list.js

```diff
@@ -32,7 +32,9 @@ export class ProductsWidget extends Component {
         this.orm = useService("orm");
     }
     get hasProducts() {
-        return Object.keys(this.pos.globalState.db.product_by_id).length > 0;
+        return Object.values(this.pos.globalState.db.product_by_id).some(
+            (p) => p.available_in_pos && p.lst_price > 0
```

Comment:
Because there are tips and discounts that have a price of 0 and are preventing the demo data generation "wizard" (buttons) from behind displayed since they are available in pos as a normal product. This criterion is really simple and allows to check if there is at least one real product, otherwise display the demo generation wizard.

Reply:
I don't see why would anybody have the point of sale app for selling products for free. But in any case, if such case exists, those people can create an useless product with a price, and never use it...

Reply:
As I said, such approximation is reasonable. It's not worth it to take into account those users. We can mark this as resolved.